packages/log4j2-hotpatch/0001-Log-to-stderr-instead-of-stdout.patch

```diff
@@ -0,0 +1,35 @@
+From 61921bad328e3b12305fd91162c30e9a54bedfbf Mon Sep 17 00:00:00 2001
+From: Stewart Smith <trawets@amazon.com>
+Date: Thu, 16 Dec 2021 07:54:47 -0800
+Subject: [PATCH] Log to stderr instead of stdout
+
+Signed-off-by: Stewart Smith <trawets@amazon.com>
+---
+ src/main/java/Log4jHotPatch.java | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/src/main/java/Log4jHotPatch.java b/src/main/java/Log4jHotPatch.java
+index 85cabd4..a45d934 100644
+--- a/src/main/java/Log4jHotPatch.java
++++ b/src/main/java/Log4jHotPatch.java
+@@ -58,7 +58,7 @@ public class Log4jHotPatch {
+ 
+   private static void log(String message) {
+     if (verbose) {
+-      System.out.println(message);
++      System.err.println(message);
+     }
+   }
+ 
+@@ -260,7 +260,7 @@ public class Log4jHotPatch {
+         log("Patching all JVMs!");
+       }
+     } else if (args.length == 1 && ("-h".equals(args[0]) || "-help".equals(args[0]) || "--help".equals(args[0]))) {
+-      System.out.println("usage: Log4jHotPatch [<pid> [<pid> ..]]");
++      System.err.println("usage: Log4jHotPatch [<pid> [<pid> ..]]");
```

Comment:
Is it common for `-help` output to be on stderr?

Reply:
It's not. Not sure if there's a good reason for this - @stewartsmith ?